gkeys/gkeys/lib.py

```diff
@@ -273,6 +273,22 @@ def list_keys(self, keydir, fingerprint=None, colons=False):
         self.logger.info('GPG return code: ' + str(result.returncode))
         return result
 
+    def send_keys(self, gkey):
+        '''Send gkey to keyserver
+        @param gkey: the gkey to be sent to the server
+        @return: GKEY_CHECK instance
+        '''
+        self.config.defaults['gpg_defaults'].append('--no-permission-warning')
+        self.set_keyserver()
+        self.set_keydir(gkey.keydir, 'send-keys', reset=True)
+        self.set_keyring('pubring.gpg', 'send-keys', reset=False)
+        self.set_keyseedfile(refresh=True)
+        self.logger.debug("LIB: send-keys, gkey: %s" % str(gkey))
+        self.logger.debug("** Calling runGPG with Running 'gpg %s --send-keys' for: %s"
+            % (' '.join(self.config.get_key('tasks', 'send-keys')), str(gkey)))
+        result = self.runGPG(task='send-keys', inputfile='')
```

Comment:
But isn't `send-key` in libs used for each key separately [here](https://github.com/gentoo/gentoo-keys/pull/50/files#diff-4c2a2fa0b979e98b8ad32a8dea1dfd8bR209)? So, isn't it certain that it will be used for only one key each time?